src/cross_module_fn.c

```diff
@@ -127,21 +127,8 @@ error_no_default_fn_community(void)
 {
 	ereport(ERROR,
 			(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-			 errmsg("functionality not supported under the current license \"%s\", license",
-					ts_guc_license_key),
-			 errhint(
-				 "Upgrade to a Timescale-licensed binary to access this free community feature")));
-}
-
-static void
-error_no_default_fn_enterprise(void)
-{
-	ereport(ERROR,
-			(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-			 errmsg("functionality not supported under the current license \"%s\", license",
-					ts_guc_license_key),
-			 errhint("Request a trial license to try this feature for free or contact us for more "
-					 "information at https://www.timescale.com/pricing")));
+			 errmsg("functionality not supported under the current \"%s\" license", ts_guc_license),
```

Comment:
Hmm you are changing the text here and the current binary might not be upgradable when compiled with APACHE_ONLY, maybe switch text depending on whether compiled with APACHE_ONLY

Reply:
If it was compiled with APACHE_ONLY this will also change the DEFAULT_LICENSE to apache. This value will be represented in ts_guc_license as well